server/app/src/main/java/com/ams/app/dto/Requestdto.java

```diff
@@ -0,0 +1,12 @@
+package com.ams.app.dto;
+import lombok.Data;
+import java.util.List;
+
+@Data
+public class Requestdto {
+    private List<Searchrequestdto> search;
+    private Operators operator;
+    public enum Operators{
```

Comment:
Why this is needed in DTO? We can remove it 
If needed, We can have an enum in Frontend side also

Reply:
I am performing specifications using post method which can be more flexible for many operations. So created a dto for it. In the Specification function, I have used this enum to compare request and perform the operations.